rerun_py/README.md

```diff
@@ -90,6 +90,8 @@ While we try to keep the main branch usable at all times, it may be unstable occ
 pixi run -e py py-build && pixi run -e py py-test
 ```
 
+If you run into a problem, run `rm -rf .pixi` and try again.
```

Comment:
oddly specify for python unit test. might as well also write "`rm -rf target` and try again", something I get a fair share of :/